Правилник.md

```diff
@@ -6,8 +6,8 @@
 (2) По въпроси, които не са уредени изрично в този Правилник, Съветът на init Lab може да взема решения в съответствие с относимото българско и европейско законодателство.
 (3) В случай на противоречия между този Правилник и решенията на Съвета на init Lab от една страна и относимото българско и европейско законодателство от друга страна, с предимство се прилагат повелителните норми на закона.
 
-Чл. 2. (1) Този Правилник може да бъде изменян и допълван от органите на Сдружението и влиза в сила 1 месец след обявяване на промените. Официална актуална версия на Правилника е достъпна от този адрес: http://initlab.org/правилник/
-(2) Последните изменения и допълнения в този Правилник са от 28.06.2021 г.
+Чл. 2. (1) Този Правилник може да бъде изменян и допълван от органите на Сдружението и влиза в сила 1 месец след обявяване на промените. Официална актуална версия на Правилника е достъпна от този адрес: https://github.com/initLab/initLab/edit/master/%D0%9F%D1%80%D0%B0%D0%B2%D0%B8%D0%BB%D0%BD%D0%B8%D0%BA.md
```

Comment:
URL is not accessible by anonymous users. Use this instead: https://github.com/initLab/initLab/blob/master/%D0%9F%D1%80%D0%B0%D0%B2%D0%B8%D0%BB%D0%BD%D0%B8%D0%BA.md

Reply:
Fixed in d8384a6